#Erreurs courantes

#Tout d'abord, supprimez tout autre code de votre main.pydÃ©poser. Copiez chaque extrait de code ci-dessous dans main.pyen cliquant sur l'icÃ´ne de copie en haut Ã  droite de chaque case de code. Ensuite, frappez runet voyez quelles erreurs se produisent. Corrigez les erreurs et appuyez sur runencore une fois jusqu'Ã  ce que vous soyez sans erreur. Clique sur le ğŸ‘€ Answerpour comparer votre code au bon code.
#Erreur de nom

#ğŸ‘‰ Quelle est l'erreur ici ?

counter = 0
while true:
  answer = int(input("Enter a number: "))
  print("Adding it up!")
  counter += answer
  print("Current total is", counter)
  addAnother = input("Add another? ")
  if addAnother == "no":
    break
print("Bye!")

while truedoit Ãªtre while True.

#Remarquez que lorsque vous remplacez le Â« t Â» minuscule par un Â« T Â» majuscule, la couleur du mot change car Replit le reconnaÃ®t dÃ©sormais comme une boucle boolÃ©enne.
#Erreur de syntaxe

#ğŸ‘‰ Et celui-ci ? Que se passe-t-il lorsque vous frappez run?

counter = 0
while True:
  answer = int(input("Enter a number: "))
  print("Adding it up!")
  counter += answer
  print("Current total is", counter)
addAnother = input("Add another? ")
if addAnother == "no":
  break
print("Bye!")

#Notez que le message d'erreur indique l'erreur de syntaxe "rupture en dehors de la boucle". Remarquez-vous comment les trois derniÃ¨res lignes avant le bas printLes instructions ne font pas partie de la boucle car elles ne sont pas correctement indentÃ©es (regardez les lignes verticales) ?

#Mettez en surbrillance ces trois lignes de code et appuyez sur tabtouchez une fois pour indenter ce code afin qu'il soit Ã  l'intÃ©rieur de la boucle. 